sdk/identity/identity/src/credentials/cliCredential.ts

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { TokenCredential, GetTokenOptions, AccessToken } from "@azure/core-http";
+import { createSpan } from "../util/tracing";
+import { AuthenticationErrorName } from "../client/errors";
+import { CanonicalCode } from "@opentelemetry/types";
+import { logger } from "../util/logging";
+import {
+  CredentialClient,
+  CliCredentialClient,
+  CliCredentialOptions
+} from "../client/CliCredentialClient";
+
+/**
+ * Provides the user access token and expire time
+ * with azure cli command "az account get-access-token" in powershell.
+ */
+export class CliCredential implements TokenCredential {
```

Comment:
This class is called `AzureCliCredential` in the other language implementation, we should be consistent unless we have a specific reason to differ.